package.json

```diff
@@ -2,12 +2,13 @@
   "name": "rollup-starter-project",
   "version": "1.0.0",
   "description": "Sample project for packages built with rollup.",
-  "main": "dist/rollup-starter-project.js",
-  "jsnext:main": "lib/index.js",
+  "main": "dist/rollup-starter-project.umd.js",
+  "jsnext:main": "dist/rollup-starter-project.es6.js",
   "scripts": {
-    "build": "mkdir -p dist && rollup lib/index.js | babel -m umd --module-id rollupStarterProject -o dist/rollup-starter-project.js",
-    "pretest": "eslint lib test",
-    "test": "mocha --compilers js:babel/register --recursive test",
+    "prebuild": "eslint lib test",
+    "build": "rollup -c rollup.umd.config.js && rollup -c rollup.es6.config.js",
```

Comment:
I don't like that this build the package twice. Ideally we'd be able to specify this configuration somehow in `rollup.config.js`.